client/src/components/payments/checkoutForm.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from "react";
+import { pay } from '../../api/payments';
```

Comment:
nitpic: instead of using a relative path from this file, can we map it from src (don't know if that is set up already)